libs/components/src/lib/listbox/listbox.scss

```
@@ -26,11 +26,14 @@
 	@include appearance.appearance;
 
 	position: relative;
+	display: flex;
+	flex-direction: column;
 	padding: 4px;
 	background-color: var(#{appearance.get-appearance-token(fill)});
 	border-radius: 6px;
 	box-shadow: inset 0 0 0 1px var(#{appearance.get-appearance-token(outline)});
 	contain: paint;
+	gap: 4px;
```

```suggestion
	gap: 2px;
```
why did you change it to 4?

---

Consist with our unit